.gitignore

```diff
@@ -1,2 +1,5 @@
 /target
 **/*.rs.bk
+
+# intellij
```

Comment:
I'm fine with this but I recommend you to setup a global .gitignore in your system